site/content/en/docs/Examples/supertuxkart.md

```diff
@@ -0,0 +1,97 @@
+---
+
+## Prerequisite
+
+To get started, ensure the following prerequisites are met:
+
+  - You have a running Kubernetes cluster.
+
+  - Agones is installed on your cluster. Refer to the [Agones guide](https://agones.dev/site/docs/installation/install-agones/) for the instructions.
+
+  - The SuperTuxKart client downloaded for gameplay. Download it from [SuperTuxKart](https://supertuxkart.net/Main_Page)
+
+  - (Optional) Review {{< ghlink href="examples/supertuxkart" >}}SuperTuxKart code{{< /ghlink >}} to see the details of this example.
+
+## Create a Fleet
+
+Let's create a Fleet using the following command:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/examples/supertuxkart/fleet.yaml
+```
+
+You should see a successful output similar to this :
+
+```
+fleet.agones.dev/supertuxkart created
+```
+
+This has created a Fleet record inside Kubernetes, which in turn creates two ready [GameServers]({{< ref "/docs/Reference/gameserver.md" >}})
+that are available to be allocated for a game session.
+
+```bash
+kubectl get fleet
+```
+It should look something like this:
+
+```
+NAME           SCHEDULING   DESIRED   CURRENT   ALLOCATED   READY   AGE
+supertuxkart   Packed       2         2         0           2       55s
+```
+
+You can also see the GameServers that have been created by the Fleet by running `kubectl get gameservers`,
+the GameServer will be prefixed by `supertuxkart`.
+
+```
+NAME                             STATE       ADDRESS        PORT   NODE                                  AGE
+supertuxkart-xfw2g-bwwkb         Ready       34.82.158.69   7421   gke-agon-default-pool-f18c8e90-1f9k   103s
+supertuxkart-xfw2g-skdnf         Ready       34.82.158.69   7585   gke-agon-default-pool-f18c8e90-1f9k   103s
+```
+
+For the full details of the YAML file head to the [Fleet Specification Guide]({{< ref "/docs/Reference/fleet.md" >}})
+
+{{< alert title="Note" color="info">}} The game servers deployed from a `Fleet` resource will be deployed in the same namespace. The above example omits specifying a namespace, which implies both the `Fleet` and the associated `GameServer` resources will be deployed to the `default` namespace. {{< /alert >}}
+
+## Viewing GameServer Logs
+
+For troubleshooting or to check how your game servers are running, you can look at the logs of a specific pod using:
+
+```bash
+kubectl logs -f <supertuxkart-game-server-pod-name>
+```
+
+## Connect to the Game Server
+
+After allocating a GameServer from the fleet and obtaining its status and IP, you're ready to connect and play. Here’s how to use the server IP and port to join the game with the SuperTuxKart client:
+
+**Launch SuperTuxKart**: Start the SuperTuxKart client you downloaded earlier by running the executable for your operating system ([documentation]https://supertuxkart.net/FAQ)
```

Comment:
Markdown issue
```suggestion
**Launch SuperTuxKart**: Start the SuperTuxKart client you downloaded earlier by running the executable for your operating system ([documentation](https://supertuxkart.net/FAQ))
```

I think will fix it. If you need local preview: `make site-server`

Reply:
I've fixed the link issue